app/lib/frontend/templates/package.dart

```diff
@@ -292,12 +260,57 @@ String renderPkgShowPage(
   );
 }
 
-List<Map<String, String>> _tabsData(
+/// Renders the `views/pkg/tabs.mustache` template.
+String renderPkgTabs(List<Tab> tabs) {
+  final tabsData = tabs.map((t) => t.toMustacheData()).toList();
+  // active: the first one with content
+  for (int i = 0; i < tabs.length; i++) {
+    if (tabs[i].contentHtml != null) {
+      tabsData[i]['active'] = '-active';
```

Comment:
Couldn't this be called `classes`?

Reply:
It is hard, because we use `markdown-body` in the content classes (if `isMarkdown` is set), while `-active` is used in both the headers and the content.
I'll make an effort in a subsequent PR to make them more uniform.